content/docs/command-reference/data/index.md

```diff
@@ -0,0 +1,24 @@
+# data
+
+Contains a command that shows changes in data tracked by dvc:
+[status](/doc/command-reference/data/status).
```

Comment:
This matches what we have in other sections, so it makes sense. Should we err on the side of being precise about current functionality or hinting towards what we anticipate for `dvc data`? I would prefer something more generic, like "A set of commands to manage data tracked by DVC." If that's too much of a stretch for now, I'm fine leaving it as is.

Reply:
Let's change to "set of commands" when there are more subcommands. Or are they all being released together and this is PR 1 out of more?

Reply:
>  Or are they all being released together and this is PR 1 out of more?

There's just one planned for now. To be honest, I don't even see a benefit of having an index file.